src/System.Windows.Forms/src/System/Windows/Forms/Button.cs

```diff
@@ -89,17 +89,7 @@ public AutoSizeMode AutoSizeMode
             }
         }
 
-        protected override AccessibleObject CreateAccessibilityInstance()
-        {
-            return new ButtonAccessibleObject(this);
-        }
-
-        private class ButtonAccessibleObject : ControlAccessibleObject
-        {
-            public ButtonAccessibleObject(Button owner) : base(owner)
-            {
-            }
-        }
+        internal override bool SupportsUiaProviders => true;
```

Comment:
We do not need this as Control has already overridden `SupportsUiaProviders => true;`.